android/build.gradle

```diff
@@ -51,7 +70,8 @@ repositories {
 }
 
 dependencies {
-    implementation "com.facebook.react:react-native:+"  // From node_modules
+    //noinspection GradleDynamicVersion
+    api "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
api is deprecated

Reply:
Consider you have app module which uses lib1 as a library and lib1 uses lib2 as a library. Something like this: app -> lib1 -> lib2.

Now when using api lib2 in lib1, then app can see lib2 codes when using: api lib1 or implementation lib1 in the app module.

BUT when using implementation lib2 in lib1, then app can not see the lib2 codes.

src: https://stackoverflow.com/questions/44413952/gradle-implementation-vs-api-configuration